src/Components/Banana/CreateBanana.jsx

```diff
@@ -0,0 +1,43 @@
+let bananaIdNumCount = 0;
+
+function CreateBanana(prop) {
+
+    bananaIdNumCount ++;
+
+    const bananaId = bananaIdNumCount;
+
+    // This defines a new creation of the banana element
+    const newBananaImage = document.createElement("img");
+
+    // Adds attributes to the banana element
+    newBananaImage.setAttributeNS(null, 'id', `bananaId${bananaId}`);
+    newBananaImage.setAttributeNS(null, 'class', 'banana');
+    newBananaImage.setAttributeNS(null, 'alt', 'bananaPic');
+    newBananaImage.setAttributeNS(null, 'src', '../images/banana.png');
+
+    // Adds a click event to the banana element
+    newBananaImage.addEventListener('click', bananaClickHandle);
+
+    // Appends banana element to the the selected div
+    document.getElementById('imageBox').appendChild(newBananaImage);
+    
+    // Delete old bananas
+    setTimeout(() => {
+        if(document.getElementById(`bananaId${bananaId}`)) {
+            document.getElementById(`bananaId${bananaId}`).remove();
+            prop({
+                type: 'MISSED_BANANA'
+            })
+        }
+    }, 4000)
+
+    // Delete clicked bananas
+    function bananaClickHandle() {
+        document.getElementById(`bananaId${bananaId}`).remove();
+        prop({
+            type: 'CAUGHT_BANANA'
+        })
+    }
+}
+
+export default CreateBanana;
```

Comment:
So since these aren't actually React components, as in neither of these functions return JSX, I would move these out of your components folder and maybe make a new folder in your `src` folder called `utils`.  Having a `utils` folder is a common thing in these types of projects.  Then in your `utils` folder you can make like an `index.js` and instead of default exporting one function you can do regular exports on all the utility functions and import them as needed into components that need to use them.